pkg/api/types.go

```diff
@@ -196,3 +196,12 @@ type DiskInfo struct {
 	BlockSize   int64
 	ClusterSize int64
 }
+
+type ReplicaInstance struct {
```

Comment:
Why can't we directly use `Replica` in `longhorn-spdk-engine/pkg/api`?
I am thinking if there are too many replica/engine related struct definitions/structs...

Reply:
ReplicaInstance will be used for legacy and spdk volumes' replica directories or spdk replica. I think a it's more clear for them.

Reply:
I see. This is unrelated to process/spdk instances. It's actually the entries of the disk ReplicaMap. I am thinking if it's better to rename it.

Reply:
Rename to `ReplicaStorageInstance`